engine/src/exec-interface-group.cpp

```diff
@@ -139,8 +139,6 @@ void MCGroup::SetRadioBehavior(MCExecContext& ctxt, uint32_t part, bool setting)
 	bool t_dirty;
 	t_dirty = changeflag(setting, F_RADIO_BEHAVIOR);
 
-	if (flags & F_RADIO_BEHAVIOR)
```

Comment:
@livecodepanos I'm wondering why this was ever here. Odd thing is that group default is radioBehavior true with tabGroupBehavior false so why would we be forcing tabGroupBehavior true when setting radioBehavior!